Tests/SentryTests/Integrations/Session/SentrySessionTrackerTests.swift

```diff
@@ -422,12 +422,15 @@ class SentrySessionTrackerTests: XCTestCase {
         
         if fixture.client.captureSessionInvocations.invocations.indices.contains(endSessionIndex) {
             let session = fixture.client.captureSessionInvocations.invocations[endSessionIndex]
-            XCTAssertFalse(session.flagInit?.boolValue ?? false)
-            XCTAssertEqual(started, session.started)
-            XCTAssertEqual(SentrySessionStatus.exited, session.status)
-            XCTAssertEqual(errors, session.errors)
-            XCTAssertEqual(started.addingTimeInterval(TimeInterval(truncating: duration)), session.timestamp)
-            XCTAssertEqual(duration, session.duration)
+            if let flagInit = session.flagInit {
+                XCTAssertFalse(flagInit.boolValue)
+            }
+            XCTAssertEqual(started, session.started, "Expected \(started) but got \(session.started)")
```

Comment:
I'm a bit confused. `XCTAssertEqual` already has a default error message showing the expected and actual values, right? See for example https://github.com/getsentry/sentry-cocoa/runs/7766682459?check_suite_focus=true: 

```
testForegroundWithError, XCTAssertEqual failed: ("Optional(2)") is not equal to ("nil")
testKillAppWithoutNotificationsAndNoCrash_EndsWithAbnormalSession, XCTAssertEqual failed: ("2001-01-01 00:00:00 +0000") is not equal to ("2001-01-01 00:00:02 +0000")
```

etc

Reply:
Ugh yes of course, I got too fixated on the lines above those that only showed the assertion that failed 🤦🏻